internal/factsengine/gatherers/sbddump.go

```diff
@@ -0,0 +1,50 @@
+package gatherers
+
+import (
+	"fmt"
+	"strings"
+
+	log "github.com/sirupsen/logrus"
+	"github.com/trento-project/agent/internal/utils"
+)
+
+const (
+	SBDDumpFactKey = "SBD_dump"
+)
+
+type SBDDumpGatherer struct {
+	executor CommandExecutor
+}
+
+func NewSBDDumpGatherer() *SBDDumpGatherer {
+	return &SBDDumpGatherer{
+		executor: Executor{},
+	}
+}
+
+func (s *SBDDumpGatherer) Gather(factsRequests []FactRequest) ([]Fact, error) {
+	facts := []Fact{}
+	log.Infof("Starting %s facts gathering process", SBDDumpFactKey)
+
+	for _, factReq := range factsRequests {
+		args := strings.Split(factReq.Argument, ":")
```

Comment:
We should do some error handling here to check the arguments length, or at least put some comment saying that this is temporary solution, and that the argument usage will change soonish